app-misc/dateutils/dateutils-0.4.6.ebuild

```diff
@@ -19,6 +27,11 @@ RDEPEND="!sys-fabric/dapl"
 
 PATCHES=( "${FILESDIR}"/${P}-unportable-sys-sysctl_h.patch )
 
+src_configure() {
+	[ "${PV}" = 9999 ] && autoreconf -i
```

Comment:
1. Please use [**e**autoreconf](https://devmanual.gentoo.org/eclass-reference/autotools.eclass/index.html) because it will die on failure, and handles all kinds of complexities for us. You probably won't need an argument for it.

You'll also need to then inherit the eclass.

2. We [use](https://unix.stackexchange.com/questions/306111/what-is-the-difference-between-the-bash-operators-vs-vs-vs) [[ ]] rather than [ ].